src/api/versions/v1/services/websocket-service.ts

```
@@ -119,28 +119,32 @@ export class WebSocketService {
+    console.debug(
+      `%cReceived message from user ${user.getName()}:\n` +
+        binaryReader.preview(),
+      "color: green;"
+    );
+
+    const typeId = binaryReader.unsignedInt8();
 
     switch (typeId) {
       case WebSocketType.PlayerIdentity: {
-        return this.handlePlayerIdentityMessage(user, payload);
+        return this.handlePlayerIdentityMessage(user, binaryReader);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)